address_book/address_book.py

```diff
@@ -19,22 +19,22 @@ def find(self, name):
     def delete(self, name):
         del self.data[name]
 
-    def get_upcoming_birthdays(self) -> list[dict[str, str]]:
+    def get_upcoming_birthdays(self, days) -> list[dict[str, str]]:
```

Comment:
```suggestion
    def get_upcoming_birthdays(self, days: int) -> list[dict[str, str]]:
```
@NotJod563  It would be great to have typing for the argument